src/main/java/crystal/Ui.java

```diff
@@ -0,0 +1,179 @@
+package crystal;
+
+import crystal.task.Task;
+import crystal.task.Todo;
+import crystal.task.Deadline;
+import crystal.task.Event;
+
+import java.util.Scanner;
+
+/**
+ * Represents the Ui task.
+ *
+ */
+public class Ui {
+    Scanner sc = new Scanner(System.in);
+
+    /**
+     * Return the next Line in the file
+     *
+     */
+    public String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Prints the welcome message.
+     *
+     */
+    public void showWelcome() {
+        System.out.println(" ____________________________________________________________");
+        System.out.println(" Hi! I am CRYSTAL.\n How may I be of assistance?");
+        System.out.println(" ____________________________________________________________");
+    }
+
+    /**
+     * Prints the error message.
+     * @param e Exception
+     */
+    public void showError(CrystalException e) {
+        System.out.println(" ____________________________________________________________");
+        System.out.println(e.getMessage());
+        System.out.println(" ____________________________________________________________");
+    }
+
+    /**
+     * Prints the error message if the file failed to load.
+     *
+     */
+    public void showLoadingError() {
+        System.out.println("Load Error");
+    }
+
+    /**
+     * Prints the list.
+     * @param task tasklist
+     */
+    public void printList(TaskList task) {
+        System.out.println(" ____________________________________________________________");
+        System.out.println("Here are your current tasks:");
+        for (int i = 0; i < task.size(); i++) {
+            System.out.println(i + 1 + ". " + task.get(i));
+        }
+        System.out.println(" ____________________________________________________________");
+    }
+
+    /**
+     * Prints the unmark task message.
+     * @param task tasklist.
+     * @param num The task number to be unmarked
+     *
+     */
+    public void printUnmark(TaskList task, int num) {
+        Task unmarkTask = task.get(num-1);
+        System.out.println(" ____________________________________________________________");
+        System.out.println("Alright, I've marked this task as not done: ");
+        unmarkTask.isDone = false;
+        System.out.println(unmarkTask.toString());
+        System.out.println(" ____________________________________________________________");
```

Comment:
I like your output ui style !!!  It looks very neat and crystal clear ! :star_struck::star_struck::star_struck:

Reply:
Thank you!